examples/aws-testing/cluster.lokocfg

```diff
@@ -30,8 +30,11 @@ cluster "aws" {
   dns_zone         = var.dns_zone
   dns_zone_id      = var.route53_zone_id
   ssh_pubkeys      = var.ssh_public_keys
-  worker_count     = var.workers_count
-  worker_type      = var.workers_type
+  worker_pool "my-worker-pool-name" {
+    instance_type = var.workers_type
```

Comment:
Indentation.

Reply:
same here, indentation seems to be fine.

Reply:
That's right, but maybe an empty line before the `worker_pool` section would help.

Reply:
ok

Reply:
```diff
diff --git a/examples/aws-testing/cluster.lokocfg b/examples/aws-testing/cluster.lokocfg
index ef687e71..1e9663a7 100644
--- a/examples/aws-testing/cluster.lokocfg
+++ b/examples/aws-testing/cluster.lokocfg
@@ -32,8 +32,8 @@ cluster "aws" {
   ssh_pubkeys      = var.ssh_public_keys
   worker_pool "my-worker-pool-name" {
     instance_type = var.workers_type
-    count = var.workers_count
-    ssh_pubkeys = var.ssh_public_keys
+    count         = var.workers_count
+    ssh_pubkeys   = var.ssh_public_keys
   }
 }
 
```